lib/xcodebuild.js

```diff
@@ -183,6 +185,10 @@ class XcodeBuild {
       args = [buildCmd, testCmd];
     }
 
+    if (this.allowProvisioningDeviceRegistration) {
+      args.push('-allowProvisioningUpdates', '-allowProvisioningDeviceRegistration');
```

Comment:
Could you leave a comment about `allowProvisioningUpdates` why this option provides both parameter?

>     -allowProvisioningDeviceRegistration                     Allow xcodebuild to register your destination device on the developer portal if necessary. This flag only takes effect if -allowProvisioningUpdates is also passed.

Reply:
I've added it already.